infra/gcb/build_and_run_coverage.py

```diff
@@ -175,9 +199,35 @@ def get_build_steps(project_dir):
               UPLOAD_URL_FORMAT.format(
                   project=project_name, type='logs', date=report_date),
           ],
-      },
-  ])
+      }
+  )
 
+  # Update the latest report information file for ClusterFuzz.
+  latest_report_info_url = build_project.get_signed_url(
+      LATEST_REPORT_INFO_URL.format(project=project_name),
+      method='PUT',
+      content_type='application/json')
+  latest_report_info_body = json.dumps(
+      {
+          'fuzzer_stats_dir': upload_fuzzer_stats_url,
+          'html_report_url': HTML_REPORT_URL_FORMAT.format(
+              project=project_name, date=report_date, platform=PLATFORM),
+          'report_summary_path': os.path.join(
+              upload_report_url, PLATFORM, 'summary.json'),
```

Comment:
would adding the date here be useful?

Reply:
Yeah, having a date in there would be useful. Thanks for the suggestion!